views/index.ejs

```diff
@@ -47,6 +47,9 @@
             });
           ga('require', 'urlChangeTracker');
     </script>
+    <noscript>
+     Javascript is disabled. Please enable javascript and refresh the page.
```

Comment:
Totally agree @elisechant .  We will need to style this and provide proper instructions.  I just wanted to do a quick experiment for a user who I believe has js disabled.